test/Microsoft.AspNetCore.Diagnostics.Tests/ExceptionHandlerTest.cs

```diff
@@ -176,6 +176,48 @@ public class ExceptionHandlerTest
         }
 
         [Fact]
+        public async Task Reexecute_RequestWithQueryString()
+        {
+            var expectedStatusCode = 432;
+            var destination = "/location";
+            var builder = new WebHostBuilder()
+                .Configure(app =>
+                {
+                    app.UseStatusCodePagesWithReExecute("/errorPage", "?id={0}");
+
+                    app.Map(destination, (innerAppBuilder) =>
+                    {
+                        innerAppBuilder.Run(async (httpContext) =>
+                        {
+                            await Task.Delay(1);
```

Comment:
Just so that the delegate was async and VS wouldn't complain, I've come up with a more useful thing to do there though.